### PR TITLE
First class flows

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: lwt/mirage_flow_lwt.mli [hunk 1]
```diff
@@ -142,3 +142,27 @@ module F: sig
...
+(** [create (module M) t name] is the flow representing [t] using the
+    function defined in [M]. *)
+
+val pp: t Fmt.t
+(** [pp] is the pretty-printer for IO flows. *)
+
+val forward: ?verbose:bool -> src:t -> dst:t -> unit Lwt.t
+(** [forward ?verbose ~src ~dst] forwards writes from [src] to
+    [dst]. Block until either [src] or [dst] is closed. If [verbose]
+    is set (by default it is not), show the raw flow in debug mode,
```
Comment (avsm) [inline lwt/mirage_flow_lwt.mli]: what does "show the raw flow in debug mode" mean?  You mean print to a logger?

File: lwt/mirage_flow_lwt.mli [hunk 2]
```diff
@@ -142,3 +142,27 @@ module F: sig
...
+    otherwise just show the lenght. *)
```
Comment (avsm) [inline lwt/mirage_flow_lwt.mli]: s/lenght/length

File: lwt/mirage_flow_lwt.mli [hunk 3]
```diff
@@ -142,3 +142,27 @@ module F: sig
...
+
+val proxy: ?verbose:bool -> t -> t -> unit Lwt.t
+(** [proxy ?verbose x y] is the same as [forward x y <*> forward y
+    x]. Block until both flows are closed. If [verbose] is set (by
+    default it is not), show the raw flow in debug mode, otherwise
+    just show the lenght. *)
```
Comment (avsm) [inline lwt/mirage_flow_lwt.mli]: same typo as above

**Review by avsm — CHANGES_REQUESTED** [review]

**Comment by samoht** [comment]
Thanks, I've force-pushed doc fixes.

**Pull request merged**